packages/as-proba/assembly/probability/binomial.ts

```diff
@@ -43,10 +43,15 @@ export class Binomial extends Sampler implements Drawer {
   }
 
   /**
-   * Draws a number.
+   * Draws a number from the binomial distribution
+   *
+   * @remarks
```

Comment:
I don't see any _m in the functio nsignature. is it a privateRemarks for the contributor ?

Reply:
You are right.

Btw, is it normal that the _m attribute is neither set in the code nor by the user?

Reply:
nope. can you analyze this ?

Reply:
I opened the issue #153 to solve it in another pr